<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DrinksWidget</class>
 <widget class="QWidget" name="DrinksWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>271</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame {
	border:none;
	border-right: 4px solid rgb(35,35,35);
	border-right: 2px solid rgb(128,128,128);
}

QScrollBar {
    background: rgba(0,0,0,0);
	margin: 3px;
}
QScrollBar::handle {
    background: rgb(77,77,77);
	border: 1px solid rgb(55,55,55);
	border-radius: 4px;
}
 QScrollBar::add-line,  QScrollBar::sub-line {
     border: none;
	background: rgba(0,0,0,0);
 }

QListView {
	background: rgb(35,35,35);
	color: rgb(209,209,209);
	border-right: 1px solid  rgb(108,108,108);
}

QListView::item:selected {
    background: rgb(80,80,80);
	border-right: 1px solid rgba(0,0,0,0);
	margin-right: -10px;
}

QTreeView::item:selected {
    background: rgb(80,80,80);
}

QTreeView {
	background: rgb(35,35,35);
	color: rgb(209,209,209);
	border-right: 1px solid  rgb(108,108,108);
}

QTreeView::item:selected {
	border-right: 1px solid  rgb(108,108,108);
    background: rgb(80,80,80);
}

QLineEdit {
	background: rgb(44,44,44);
	color: rgb(230,230,230);
	border: none;
	border-bottom: 1px solid  rgb(118,118,118);
}

QToolButton {
	background: rgb(44,44,44);
	color: rgb(230,230,230);
	border: none;
	border-bottom: 1px solid  rgb(118,118,118);
	border-right: 1px solid  rgb(118,118,118);
}
QPushButton {
	background: rgb(35,35,35);
	color: rgb(209,209,209);
	border: none;
	border-right: 1px solid  rgb(108,108,108);
}

QPushButton#pushButtonAdd {
	border-bottom: 1px solid  rgb(108,108,108);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,0,1,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEditSearchDrink">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>35</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Filter drinks...</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButtonSearch">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../resource/gamewindow/gamewindow.qrc">
              <normaloff>:/gamewindow/magnifyingglass.png</normaloff>:/gamewindow/magnifyingglass.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="listViewDrinks">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="selectionRectVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listViewPlayers">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="selectionRectVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonRemove">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeView" name="treeViewDrinkRound">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetExistingDrinks">
        <widget class="QWidget" name="page">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listViewDrinks</tabstop>
  <tabstop>pushButtonRemove</tabstop>
  <tabstop>pushButtonAdd</tabstop>
 </tabstops>
 <resources>
  <include location="../../resource/gamewindow/gamewindow.qrc"/>
 </resources>
 <connections/>
</ui>
